webapp/app/plugins/pluginpanel.html

```diff
@@ -141,12 +141,21 @@
       <ul class="content-spacer">
         <li class="small">
           <label>Job:</label>
-          <a title="Filter jobs like these"
-             href="{{buildbotJobnameHref}}"
+          <a title="Filter by job data strings (or buildername)"
+             href="{{jobSearchStrHref}}"
              prevent-default-on-left-click
-             ng-click="filterByBuildername(buildbotJobname)"
-             copy-value="{{buildbotJobname}}">
-            {{buildbotJobname}}</a>
+             ng-click="filterByJobSearchStr(jobSearchStr)"
+             copy-value="{{jobSearchStr}}">
+            {{jobSearchStr}}</a>
+        </li>
+        <li class="small">
```

Comment:
allow filtering by the specific job ``signature`` which is more accurate.  However, in discussions with sheriff's, they want both strings and this.